src/viser/client/src/WebsocketMessages.ts

```diff
@@ -994,6 +994,15 @@ export interface ThemeConfigurationMessage {
       ]
     | null;
 }
+/** Message from server->client carrying line segments information.
+ *
+ * (automatically generated)
+ */
+export interface LineSegmentsMessage {
+  type: "LineSegmentsMessage";
+  name: string;
+  props: { points: Uint8Array; line_width: number; colors: Uint8Array };
```

Comment:
I might be mistaken but should this be `points: Float32Array;`

Reply:
Ah! `Uint8Array` is (for now) actually expected: when we serialize numpy arrays we strip the shape + dtype information, so all arrays just become flat uint8 buffers.